modules/ROOT/pages/amazon/amazon-dynamodb-connector-reference.adoc

```diff
@@ -1,1190 +1,1167 @@
-| Configuration | String | The name of the configuration to use. | |x
-| Key Condition Expression a| String |  The condition that specifies the key value(s) for items to be retrieved by the Query action. |  |
-| Key Conditions a| Object |  This is a legacy parameter. Use KeyConditionExpression instead. |  |
-| Scan Index Forward a| Boolean |  Specifies the order for index traversal: If true (default), the traversal is performed in ascending order; if false, the traversal is performed in descending order. |  false |
+| Configuration | String | The name of the configuration to use. | | x
+| Key Condition Expression a| String |  The condition that specifies the key value(s) for items to be retrieved by the Query action. |  | 
+| Key Conditions a| Object |  This is a legacy parameter. Use KeyConditionExpression instead. |  | 
+| Scan Index Forward a| Boolean |  Specifies the order for index traversal: If true (default), the traversal is performed in ascending order; if false, the traversal is performed in descending order. |  false | 
 | Select a| Enumeration, one of:
 
 ** ALL_ATTRIBUTES
 ** ALL_PROJECTED_ATTRIBUTES
 ** SPECIFIC_ATTRIBUTES
-** COUNT |  The attributes to be returned in the result. |  |
-| Attribute Names a| Object |  |  |
-| Attribute Values a| Object |  |  |
-| Table Name a| String |  |  |x
-| Attributes To Gets a| Array of String |  |  |
+** COUNT |  The attributes to be returned in the result. |  | 
+| Attribute Names a| Object |  |  | 
+| Attribute Values a| Object |  |  | 
+| Table Name a| String |  |  | x
+| Attributes To Gets a| Array of String |  |  | 
 | Conditional Operator a| Enumeration, one of:
 
 ** AND
-** OR |  |  |
-| Consistent Read a| Boolean |  |  false |
-| Exclusive Start Key a| Object |  |  |
-| Filter Expression a| String |  |  |
-| Index Name a| String |  |  |
-| Limit a| Number |  |  |
-| Projection Expression a| String |  |  |
-| Condition Filter a| Object |  |  |
+** OR |  |  | 
+| Consistent Read a| Boolean |  |  false | 
+| Exclusive Start Key a| Object |  |  | 
+| Filter Expression a| String |  |  | 
+| Index Name a| String |  |  | 
+| Limit a| Number |  |  | 
+| Projection Expression a| String |  |  | 
+| Condition Filter a| Object |  |  | 
 | Return Consumed Capacity a| Enumeration, one of:
 
 ** INDEXES
 ** TOTAL
-** NONE |  |  |
-| Target Variable a| String |  The name of a variable where the output of this operation is stored |  |
-| Target Value a| String |  An expression to evaluate against the operation's output and the outcome of that expression is stored in the target variable. |  `#[payload]` |
-| Reconnection Strategy a| * reconnect
-* reconnect-forever |  A retry strategy in case of connectivity errors |  |
+** NONE |  |  | 
+| Target Variable a| String |  The name of a variable to store the operation's output. |  | 
+| Target Value a| String |  An expression to evaluate against the operation's output and store the expression outcome in the target variable |  `#[payload]` | 
```

Comment:
An expression to evaluate against the operation's output. The evaluation outcome is stored in the target variable.